### DISCUSSION TITLE
Setting the refresh_interval to None from the python SDK

### DISCUSSION THREAD

**Discussion by pachuc** [looker-open-source/sdk-codegen#1029] [Q&A] [answered]
Hi, I am trying to use the `update_dashboard` function from the python sdk to set the `refresh_interval` of my dashboards to `None`. I see that dashboards that do not have a refresh_interval are represented by `None`. I also see that `update_dashboard` allows me to change the `refresh_interval` value to a different string value, i.e `"30 minutes"` to `"45 minutes"`. However, it does not seem to let me set this value to None. I think this is an SDK bug, where it expects to get a string value, but there is no string value that represents 'no interval'... Is this correct?

**Comment by rollyjoel** [comment, marked as answer]
maybe an empty string? does setting it to `None` leave it as the original value? Another option is `EXPLICIT_NULL` [like this](https://github.com/looker-open-source/sdk-codegen/blob/9b987b2c2f24e4a43d6ff25921c5f46804b7702a/python/tests/integration/test_methods.py#L68)

**Reply by pachuc** [reply]
Thanks Joel! The `EXPLICIT_NULL` is exactly what I needed!!
